simlib/mixnet-sims/Cargo.toml

```diff
@@ -16,10 +16,9 @@ serde_json = "1.0.132"
 tracing = "0.1.40"
 tracing-subscriber = { version = "0.3", features = ["json", "env-filter", "tracing-log"] }
 netrunner = { path = "../netrunner" }
-nomos-tracing = { git = "https://github.com/logos-co/nomos-node.git", rev = "cc5fef6" }
-nomos-mix = { git = "https://github.com/logos-co/nomos-node", rev = "e095964", package = "nomos-mix" }
-nomos-mix-message = { git = "https://github.com/logos-co/nomos-node", rev = "e095964", package = "nomos-mix-message" }
+nomos-tracing = { git = "https://github.com/logos-co/nomos-node.git" }
+nomos-mix = { git = "https://github.com/logos-co/nomos-node", package = "nomos-mix", branch = "missing-covertraffic-bounds" }
+nomos-mix-message = { git = "https://github.com/logos-co/nomos-node", package = "nomos-mix-message", branch = "missing-covertraffic-bounds" }
```

Comment:
Ill remove the branch once [this](https://github.com/logos-co/nomos-node/pull/914) is merged